File: Examples/AspNetMvcCS/Controllers/DocumentViewerController.UsingLayout.cs
using System.Web.Mvc;
using GleamTech.DocumentUltimate.AspNet.UI;

namespace GleamTech.DocumentUltimateExamples.AspNetMvcCS.Controllers
{
    public partial class DocumentViewerController
    {
        public ActionResult UsingLayout()
        {
            var documentViewer = new DocumentViewer
            {
                Width = 960,
                Height = 720,
                Resizable = true,
                Document = "~/App_Data/ExampleFiles/Default.pdf"
            };

            return View(documentViewer);
        }
    }
}

File: Examples/AspNetMvcVB/Controllers/DocumentViewerController.UsingLayout.vb
Imports GleamTech.DocumentUltimate.AspNet.UI

Namespace Controllers
    Partial Public Class  DocumentViewerController
        Inherits Controller

        Public Function UsingLayout() As ActionResult
            Dim documentViewer = New DocumentViewer() With {
                    .Width = 960,
                    .Height = 720,
                    .Resizable = True,
                    .Document = "~/App_Data/ExampleFiles/Default.pdf"
                    }

            Return View(documentViewer)
        End Function

    End Class
End Namespace
